src/main/java/com/alkemy/ong/service/impl/SlideServiceImpl.java

```diff
@@ -68,6 +70,21 @@ public Long saveSlide(SlideDto dto) {
 		return slidesDetailDto;
 	}
 
+	@Transactional(readOnly = true)
+	@Override
+	public List<SlideDetailDto> getAllSlidesbyOrg(Long idOrg) {
+		List<Slide> slides = repository.findByOrg(idOrg);
+		List<SlideDetailDto> slidesDetailDto = mapper.toSlideDetailDto(slides);
+		Collections.sort(slidesDetailDto, new Comparator<SlideDetailDto>() {
```

Comment:
Could you please reformat this and replace it for this: `slidesDetailDto.sort(Comparator.comparingInt(SlideDetailDto::getOrder));`